devito/grid.py

```diff
@@ -1,49 +1,49 @@
 from collections import namedtuple
 
-from devito.tools import as_tuple
-from devito.dimension import SpaceDimension, TimeDimension, SteppingDimension
+from devito.dimension import (Dimension, SpaceDimension, TimeDimension,
+                              SteppingDimension, SubDimension)
 from devito.function import Constant
 from devito.mpi import Distributor
 from devito.parameters import configuration
-from devito.tools import ArgProvider, ReducerMap
+from devito.tools import ArgProvider, ReducerMap, as_tuple
 
 from sympy import prod
 import numpy as np
 
-__all__ = ['Grid']
+__all__ = ['SubDomain']
```

Comment:
No, that was a leftover. `Grid`  is a backend-specific concept, so conceptually:
* you shouldn't actually get it  when you do ``from devito.grid import *``
* you should rather ``from devito.core import Grid`` or ``from devito.yask import Grid``

Things are structured so that when users write ``from devito import Grid, ...`` the right type of Grid is automatically fetched

not sure if it's clear?